projects/kit/components/progress/progress-circle/progress-circle.style.less

```diff
@@ -11,94 +10,43 @@
     xxl: 8rem;
 };
 
-// TODO: delete in v4.0
-@legacy-width: {
-    @s: 2em;
-    @m: 3em;
-    @l: 4em;
-    @xl: 7em;
-};
-
-// TODO: delete in v4.0
-@legacy-track-stroke: {
-    @s: 0.5em;
-    @m: 0.375em;
-    @l: 0.375em;
-    @xl: 0.375em;
-};
-
-// TODO: delete in v4.0
-@legacy-progress-stroke: {
-    @s: 0.5em;
-    @m: 0.375em;
-    @l: 0.375em;
-    @xl: 0.375em;
-};
-
 :host {
+    --t-track-stroke: var(--tui-thickness, 0.375em);
+    --t-progress-stroke: var(--tui-thickness, 0.375em);
+
     position: relative;
     display: block;
     color: var(--tui-primary);
     transform: rotate(-90deg);
     transform-origin: center;
-    font-size: 1rem; // to use em units inside svg (Safari doesn't support rem units inside svg)
```

Comment:
**TODO**: check Safari 13-17

Reply:
**iPhone 15 Pro Max IOS 17.3:**
<img height="300" src="https://github.com/taiga-family/taiga-ui/assets/35179038/a86bf141-4f8e-409b-bd9c-cc697544a24f" />

Reply:
**iPhone 14 IOS 16.5** ✅
<img height="300" src="https://github.com/taiga-family/taiga-ui/assets/35179038/58fb69cf-1295-4347-85bb-7bdbaa34836c" />

Reply:
**iPhone 13 IOS 15.4** ❌

<img height="300" src="https://github.com/taiga-family/taiga-ui/assets/35179038/6ff11cd9-2128-4aa7-89ee-e54667d0b69b" />

Reply:
I need to revert `rem` => `em` + add `TODO`

Reply:
After this reversion https://github.com/taiga-family/taiga-ui/pull/7648/commits/3bbba0e143f6be1d00f5fa6a5fac7b13aff04634 (`rem` => `em`).

**iPhone 13 IOS 15.4** ✅
<img height="300" src="https://github.com/taiga-family/taiga-ui/assets/35179038/ce399c47-4912-423d-ba99-c0b0fda792db" />